src/components/TabsList/TabsList.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+import './TabsList.scss';
+
+interface Props {
+  tabs: Tab[];
+  activeTab: string;
+}
+
+export const TabsList: FC<Props> = ({ tabs, activeTab }) => {
+  const tabContent = tabs.find(tab => tab.id === activeTab);
+
+  return (
+    <>
+      <ul className="tabs-nav">
+        {tabs.map(tab => (
+          <li className="tabs-nav__item">
+            <NavLink to={`/tabs/${tab.id}`} key={tab.id} className="tabs-nav__link" activeClassName="active">
```

Comment:
if you have more than two attributes, move each of them on the new line

Reply:
done